### ISSUE TITLE
Dates in terminal showing as Ukrainian

### ISSUE THREAD

**Issue by cluttered** [samschott/maestral#610]
Just wasted a day trying to find out why an “ls -l” lists my dates in Ukrainian.

Turns out Maestral has set LANG and LC_ALL in my .profile to uk_UA.UTF-8 rather than the correct value of en_GB.UTF-8 .

can I ask that you please modify it to leave my settings alone, I suspect very few English/US citizens want their dates in Ukrainian and if they do they will have set it themselves already!!

**Comment by samschott** [comment]
@cluttered, Maestral does not touch your ˋ.profileˋ or modify any environment variables or settings in your environment. It has its own config files for itself which live in ˋ~/.local/config/maestralˋ and don't affect any other program.

Whatever changes you see must have come from another source.

**Comment by cluttered** [comment]
The entry seems to be from the cli install:

export LCALL=ukUA.UTF-8 ; export LANG=uk_UA.UTF-8 #Maestral CLI

**Comment by samschott** [comment]
Hm, how did you install Maestral? It is possible that someone else packaged it for your platform with a custom installation process.

**Comment by cluttered** [comment]
I downloaded the mac pkg I think or maybe brew. It was ages ago but I only just noticed the dates, not sure why, the cli was installed sometime back and this only seems to have been an issue this last few days.

Apologies to you, obviously not your fault.

Love Maestral so glad I have it sorted. I can’t abide the official Dropbox app, always demanding an update, forever ‘tweaking’ my settings to add itself in as an extension or whatever else they have decided to try and forever using the internet when it shouldn’t be.

Many thanks,

Mark.
